README.md

```diff
@@ -70,8 +70,14 @@ func main() {
 }
 ```
 
+## Minimal Socket Mode usage:
+
+See https://github.com/slack-go/slack/blob/master/examples/socketmode/socketmode.go
+
 ## Minimal RTM usage:
 
+(For most applications, Socket Mode is a better way to communicate with Slack)
```

Comment:
Please add a link to the official announcement.

Reply:
Done. Note that I couldn't link to the exact note but to the page that mentioned it. It's very prominent there though.